integration-cli/docker_cli_wait_test.go

```diff
@@ -67,16 +59,8 @@ func (s *DockerSuite) TestWaitNonBlockedExitRandom(c *check.C) {
 	out, _ := dockerCmd(c, "run", "-d", "busybox", "sh", "-c", "exit 99")
 	containerID := strings.TrimSpace(out)
 
-	status := "true"
-	var err error
-	for i := 0; status != "false"; i++ {
-		status, err = inspectField(containerID, "State.Running")
-		c.Assert(err, check.IsNil)
-
-		time.Sleep(time.Second)
-		if i >= 60 {
-			c.Fatal("Container should have stopped by now")
-		}
+	if err := waitInspect(containerID, "{{.State.Running}}", "false", 1); err != nil {
+		c.Fatal(err)
```

Comment:
up above on line 18 you showed a more friendly error msg, can we do that that too? When its not as simple as "previous cmd should have worked" I think it'll help others who come later to know what's going on if we show the "Container should have stopped by now" text.